tests/integrations/import/sst_service.rs

```diff
@@ -78,6 +78,27 @@ fn test_upload_sst() {
     assert!(send_upload_sst(&import, &meta, &data).is_err());
 }
 
+#[test]
+fn test_write_sst() {
+    let (_cluster, ctx, _, import) = new_cluster_and_tikv_import_client();
+
+    let data = vec![1; 1024];
+    let crc32 = calc_data_crc32(&data);
+    let length = data.len() as u64;
+
+    let mut meta = new_sst_meta(crc32, length);
+    meta.set_region_id(ctx.get_region_id());
+    meta.set_region_epoch(ctx.get_region_epoch().clone());
+
+    let mut keys = vec![];
+    let mut values = vec![];
+    for i in 1..10 {
+        keys.push(format!("k{}", i.to_string()).into_bytes());
+        values.push(format!("v{}", i.to_string()).into_bytes());
+    }
+    send_write_sst(&import, &meta, keys, values).unwrap();
```

Comment:
nothing is checked in this test. i expect you to

1. write the SST files
2. ingest the two SST files
3. perform `check_ingested_kvs`

Use `test_ingest_sst_without_crc32` as sample.